dts: arm: Fix warning on STM32L010XB boards

wcappelle opened the pull request:
When building for an STM32K010xB MCU, there is a deprecated properties warning, which should be fixed by this PR.

Signed-off-by: Wouter Cappelle <wouter.cappelle@crodeon.com>

erwango reviewed (changes requested):
Good catch, txs.
Change itself is fine, but please fix the commit message compliance issue reported by CI checks:
"3: UC4 Line exceeds max length (113>75): "When building for an STM32K010xB MCU, there is a deprecated properties warning, which should be fixed by this PR.""

Commit:
dts: arm: Fix warning on STM32L010XB boards

When building for an STM32K010xB MCU, there is a deprecated properties
warning, which should be fixed by this PR.

Signed-off-by: Wouter Cappelle <wouter.cappelle@crodeon.com>

wcappelle commented:
@erwango , thanks for the review. I've done the changes (after some mess with git :-p )
